server/views/premises/show.njk

```diff
@@ -0,0 +1,22 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% extends "../partials/layout.njk" %}
+
+{% set pageTitle = applicationName + " - " + name %}
+
+{% block content %}
+
+	{{ govukBackLink({
+		text: "Back",
+		href: "/premises"
+	}) }}
+
+	<main class="app-container govuk-body">
```

Comment:
I know this is used elsewhere in the template repo but we shouldn't be using a `<main>` element inside the `content` block as there is already a `<main>` element in the `content` block and [the HTML spec states there should only be one](https://html.spec.whatwg.org/multipage/grouping-content.html#the-main-element) `<main>` in the document at any time. 

We can set the classes onto that `<main>` in the `content` block by doing `{% set mainClasses = "app-container govuk-body" %}`

Do you think I should put in a PR to fix this in the template repo at some point?

Reply:
This also slipped past me in your previous PR :D

Reply:
Nice catch. Yeah, definitely worth fixing in the template repo too if you get a chance 👍